test/nodejs-bundle/circuit/helpers/test-node.js

```diff
@@ -0,0 +1,21 @@
+'use strict'
+
+const Libp2p = require('../../../../src')
+const secio = require('libp2p-secio')
+
+class TestNode extends Libp2p {
+  constructor (peerInfo, transports, muxer, options) {
+    options = options || {}
+    const modules = {
+      transport: transports,
+      connection: {
+        muxer: [muxer],
+        crypto: options.isCrypto ? [secio] : null
+      },
+      discovery: []
+    }
+    super(modules, peerInfo, null, options)
+  }
+}
+
+module.exports = TestNode
```

Comment:
The reason I'm _for now_ using a custom node and utilities is because circuit tests are a bit more involved and require controlling which transports are enabled per node. I'll refactor both the utility as well as my tests to allow for more control over how the node is setup.